.idea/workspace.xml

```diff
@@ -142,7 +148,7 @@
     <property name="RunOnceActivity.OpenProjectViewOnStart" value="true" />
     <property name="RunOnceActivity.ShowReadmeOnStart" value="true" />
     <property name="WebServerToolWindowFactoryState" value="false" />
-    <property name="last_opened_file_path" value="$PROJECT_DIR$/../react-laravel-master" />
```

Comment:
Can we confirm that ".idea/workspace.xml " should be in the project?
I have a feeling it should be in the ignore file as it contains the paths.

Reply:
**From the link below, we can ignore it only once it is removed:
https://stackoverflow.com/questions/57091527/what-exactly-is-idea-workspace-xml**